certified-connectors/Public360/apiDefinition.swagger.json

```diff
@@ -0,0 +1,575 @@
+                "description": "\"x-frame-options\":",
+                "type": "string"
+              },
+              "\"x-ms-apihub-cached-response\":": {
+                "description": "\"x-ms-apihub-cached-response\":",
+                "type": "string"
+              },
+              "\"x-powered-by\":": {
+                "description": "\"x-powered-by\":",
+                "type": "string"
+              },
+              "\"x-ua-compatible\":": {
+                "description": "\"x-ua-compatible\":",
+                "type": "string"
+              },
+              "\"x-xss-protection\":": {
+                "description": "\"x-xss-protection\":",
+                "type": "string"
+              },
+              "}": {
+                "description": "}",
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/Biz/v2/api/call/SI.Data.RPC/SI.Data.RPC/DocumentService/CreateDocument": {
+      "post": {
+        "summary": "Create Document",
+        "description": "Create Document",
+        "operationId": "CreateDocument",
+        "parameters": [
+          {
+            "name": "hosturl",
+            "in": "query",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "parameter": {
+                  "type": "object",
+                  "properties": {
+                    "CaseNumber": {
+                      "type": "string",
+                      "description": "Case Number",
+                      "title": "Case Number"
+                    },
+                    "Title": {
+                      "type": "string",
+                      "description": "Title",
+                      "title": ""
+                    },
+                    "Status": {
+                      "type": "string",
+                      "description": "Status",
+                      "title": ""
+                    },
+                    "Category": {
+                      "type": "string",
+                      "description": "Category",
+                      "title": ""
+                    },
+                    "Notes": {
+                      "type": "string",
+                      "description": "Notes",
+                      "title": ""
+                    }
+                  },
+                  "description": "parameter",
+                  "required": [
+                    "Title",
+                    "Category",
+                    "Status",
+                    "CaseNumber"
+                  ]
+                }
+              },
+              "required": [
+                "parameter"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",

```

Comment:
@Pranav2889 Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
Please review.